decidim-blogs/app/views/decidim/blogs/posts/_actions.html.erb

```diff
@@ -1,33 +1,19 @@
 <div class="blog__actions">
   <div class="blog__actions-left">
     <% if endorsements_enabled? %>
-      <%# REDESIGN_PENDING: replace toggler %>
-      <button id="endorsers_list-trigger" class="cursor-pointer" data-toggler=".is-active" data-toggle="endorsers_list-trigger endorsers_list">
-        <%= cell "decidim/endorsers_list", post %>
-      </button>
+      <%= cell "decidim/endorsers_list_button", post %>
       <%= endorsement_buttons_cell(post) %>
     <% end %>
 
-    <%= link_to "#comments", class: "button button__sm button__transparent-secondary" do %>
-      <%= t("decidim.blogs.posts.show.comments") %>
-      <%= icon "chat-1-line", class: "fill-current flex-none" %>
-    <% end %>
+    <%= cell "decidim/comments_button", nil %>
   </div>
 
   <div class="blog__actions-right">
     <%= follow_button_for(post, false, button_classes: "button button__sm button__text-secondary", text_classes: "") %>
 
-    <%# REDESIGN_PENDING: implement the share button %>
-    <button class="button button__sm button__text-secondary">
-      Compartir
-      <%= icon "share-line", class: "fill-current flex-none" %>
-    </button>
+    <%= cell "decidim/share_button", nil %>
 
-    <%# REDESIGN_PENDING: implement the flag button %>
-    <button class="button button__sm button__text-secondary">
-      Denunciar
-      <%= icon "flag-line", class: "fill-current flex-none" %>
-    </button>
+    <%= cell "decidim/report_button", post %>
```

Comment:
I added this because the designs included the report and I thoug
I added this because I saw it in the designs and thought it was the correct functionality, but it's easy to revert it: I can remove the button and remove the addition of the concern made in this [commit](https://github.com/decidim/decidim/pull/9852/commits/eb76be2c8db6f63d711e63376722082f543aba19)

Reply:
@entantoencuanto Oh, I didn't even see it on the mockup. It's a case of tunnel vision for me 😄 

Can you remove this feature from blogs? Thanks!

Reply:
Done!